modules/bridge_service_perimeter/variables.tf

```diff
@@ -16,18 +16,20 @@
 
 variable "policy" {
   description = "Name of the parent policy"
+  type        = string
 }
 
 variable "description" {
   description = "Description of the bridge perimeter"
+  type        = string
 }
 
 variable "perimeter_name" {
   description = "Name of the perimeter. Should be one unified string. Must only be letters, numbers and underscores"
+  type        = string
 }
 
 variable "resources" {
   description = "A list of GCP resources that are inside of the service perimeter. Currently only projects are allowed."
-  type        = "list"
-  default     = []
+  type        = list(string)
```

Comment:
Why are we dropping the default?

Reply:
@morgante, If we use a previous default empty list and don't define resources, we pass list formatting [here](https://github.com/ivankorn/terraform-google-vpc-service-controls/blob/fix/%235/modules/bridge_service_perimeter/main.tf#L25), but we get provider/API error 
```
Error: Error creating ServicePerimeter: googleapi: Error 400: A Service Perimeter without dry_run enabled must have a valid 'status' set, but no 'status' was supplied

  on ../../modules/bridge_service_perimeter/main.tf line 17, in resource "google_access_context_manager_service_perimeter" "bridge_service_perimeter":
  17: resource "google_access_context_manager_service_perimeter" "bridge_service_perimeter" {
``` 

![image](https://user-images.githubusercontent.com/50832464/63017248-85c68180-be9e-11e9-8c4c-6b3bda411035.png)